modules/commons/src/main/java/org/apache/synapse/commons/json/Constants.java

```diff
@@ -116,4 +116,7 @@ private Constants() {
     /*Property which holds the synapse commons json stream of payload*/
     public static final String ORG_APACHE_SYNAPSE_COMMONS_JSON_JSON_INPUT_STREAM = "org.apache.synapse.commons.json.JsonInputStream";
 
+    // Property to preserve spaces in XML -> JSON transformation
+    public static final String PRESERVE_SPACES = "PRESERVE_SPACES";
```

Comment:
Shall we make this name to include JSON. like **PRESERVE_SPACES_IN_JSON** or etc. WDYT ?

Reply:
We added this property to use as a common property for both JSON and XML.